kotlin/README.md

```diff
@@ -9,7 +9,7 @@ Fury Kotlin provides additional tests and implementation support for Kotlin type
 Fury Kotlin is tested and works with the following types:
 
 - primitives: `Byte`, `Boolean`, `Int`, `Short`, `Long`, `Char`, `Float`, `Double`, `UByte`, `UShort`, `UInt`, `UShort`, `UInt`.
--  `Byte`, `Boolean`, `Int`, `Short`, `Long`, `Char`, `Float`, `Double` works out of the box with the default java implementation.
+- `Byte`, `Boolean`, `Int`, `Short`, `Long`, `Char`, `Float`, `Double` works out of the box with the default java implementation.
```

Comment:
Could we replace all `default java implementation` with `default fury java implementation`